### PR TITLE
Spring Boot: Add configuration to reference a subscription by ID

### PR DESCRIPTION
**Pull request by otrosien**
In #384 support for creating a `NakadiClient` using the subscription id was [added](https://github.com/zalando-nakadi/fahrschein#subscribing-to-events-with-an-existing-subscription-id).

To fully support the Kubernetes CRD [EventTypeSubscription](https://nakadi.docs.zalando.net/nakadi-operator/subscription/) integration in the `fahrschein-spring-boot-starter` we should introduce a property to configure this for consumers.

See https://github.com/zalando-nakadi/fahrschein/issues/427

### CODE REVIEW

File: fahrschein-spring-boot-starter/README.md [hunk 1]
```diff
@@ -201,4 +201,6 @@ For a complete overview of available properties, they type and default value ple
 | `        │   ├── batch-flush-timeout`         | `int`              | in seconds [default value](https://nakadi.io/manual.html#/subscriptions/subscription_id/events_get*batch_flush_timeout) |
 | `        │   ├── stream-timeout`              | `int`              | in seconds [default value](https://nakadi.io/manual.html#/subscriptions/subscription_id/events_get*stream_timeout) |
 | `        │   └── max-uncommitted-events`      | `int`              | [default value](https://nakadi.io/manual.html#/subscriptions/subscription_id/events_get*max_uncommitted_events)            |
-
+| `        └── subscription`                    |                   |                                                                                                                         |
+| `        │   ├── enabled`                     | `boolean`         | `false`                                                                                                                 |
```
Comment (otrosien) [inline fahrschein-spring-boot-starter/README.md:205]: @coolguy1990 why do we ned a separate section for subscription-by-id and why do we need this enabled property?

Comment (coolguy1990) [reply fahrschein-spring-boot-starter/README.md:205]: I can remove the `enabled` part and make it directly `subscription-by-id: String`. My initial plan was to follow other sections such as `oauth`, `http`, `backoff` etc and have an `enabled` flag. 

But if think this `enabled` flag is not required, i can change this to `subscription-by-id` attribute.

**Comment by otrosien** [comment]
👍

**Review by rob-h-w — APPROVED** [review]
LGTM

**Comment by rob-h-w** [comment]
:+1:

**Pull request merged**